### ISSUE TITLE
Symbol.iterator doesn't work on HTMLOptionsCollection in IE11

### ISSUE THREAD

**Issue by kottenator** [zloirock/core-js#270]
I use `babel-cli@6.18.0` and `core-js@2.4.1` (`core-js/client/core.js` loaded externally, not bundled).

I have the following code:

```js
for (let option of select.options) {/*...*/}
```

**It raises an error in IE11** (on `HTMLOptionsCollection` and on `jQuery` collection; though it works fine on arrays).

So I have to do:

```js
for (let option of Array.from(select.options)) {/*...*/}
```

Is it the only way to enjoy `for...of` functionality in IE11? _(unfortunately, we still need to support it in our project)_.

**Comment by zloirock** [comment]
cc @loganfsmyth should it be also added?

**Comment by loganfsmyth** [comment]
Interesting, @kottenator I don't have IE available to me, could you run these and let me know what the output is?

    document.createElement('select').options instanceof HTMLOptionsCollection
    document.createElement('select').options instanceof HTMLCollection

`HTMLOptionsCollection` is supposed to be a subclass of `HTMLCollection`, so I didn't add it to the `HTMLOptionsCollection`, instead relying on inheritance. Maybe IE11 doesn't have them inherit properly? Those both result in `true` on Chrome.

**Comment by kottenator** [comment]
Sure, I'll check tomorrow (i.e. in about 12 hours 😄).

**Comment by kottenator** [comment]
Opps, a bit more than 12 hours have passed...

So here's the answer:

```
// IE11
> document.createElement('select').options instanceof HTMLOptionsCollection
Error: 'HTMLOptionsCollection' is undefined

> document.createElement('select').options instanceof HTMLCollection
false

> document.createElement('select').options instanceof HTMLSelectElement
true
```

Oh you IE11!

**Comment by loganfsmyth** [comment]
Well that's gross! I think this will be fixed with the next `core-js` release due to https://github.com/zloirock/core-js/pull/249. That also included `HTMLSelectElement` so it should cover it.

**Comment by zloirock** [comment]
Fixed in `2.5.0`.
